### DISCUSSION TITLE
Stop marking models with a warning: "Someone has submitted a claim that this uses their art in it's training data and this claim is pending review".

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [civitai/civitai#114] [General]
There are a few models that appear now with a warning: "Someone has submitted a claim that this uses their art in it's training data and this claim is pending review".
All Stable Diffusion models were trained on someone's art, including all main versions 1.4, 1.5, 2.0, 2.1. Training model on someone's art is 100% legal, copying style is 100% legal. Not only it's legal, but it also moral, it's called freedom, freedom of expression and freedom of thought, freedom to learn, and freedom to create. If you start prohibiting including someone's art in models without "consent", you need to delete all models on the site. Singling out any model is complete hypocrisy on your part.
If your standards prohibit including someone's art in training, please erase all the models and close your site.
Otherwise, remove this shameful warning.

**Comment by jptrot** [comment]
Recently noticed several notes stating "Someone has submitted a claim that this uses their art in it's training data and this claim is pending review". I urge Civitai either to A. Do not get involved in this emotional based drama, or B. remove all models and shut your site down entirely. Every single model you host on your site were trained using images form a wide variety of artist, many of whom have copyrighted images. I would THINK that the creators of this site would be aware how models are made and would know this. There is no case at all that has established training a mode on a sample of an artist's style would be a violation of anything. In fact, just the opposite is true. You can't copyright a style. How do you not know this? Please, be professional, do not get involved in emotional dramas and keep providing your service in a professional manner with slapping false and accusatory tags on people's models.

**Comment by Janca** [comment]
A site can't ignore take down request, and having too many could result from a number of things including having their domain revoked, like if their domain holder was pressured like Patreon and KickStarter were with another recent AI campaign, perhaps uploaders can use different legalese to avoid such things, like saying their model is `based on the style of x artist` and not actually display any featured works or likeness to their works. I understand how it works, and understand you need to train the concept on said artist, but again, a site cannot ignore such request for too long.

**Comment by James0MEGA** [comment]
Understand that there is no middle ground with the Anti-AI crowd: if you don't stop appeasing them you're going to get shut down, or lose so much utility you go defunct. The choice is yours, and it's right now.

**Comment by a-l-e-x-d-s-9** [comment]
I'll add relevant part of the discussion on [Reddit](https://www.reddit.com/r/StableDiffusion/comments/zv6f0o/comment/j1pbov4/?utm_source=reddit&utm_medium=web2x&context=3).
![image](https://user-images.githubusercontent.com/6518064/209552583-4501cb96-0bf3-4b21-b28d-46c9aaa395c8.png)

So do you consider copying style as morally wrong, but models that copy more than one style are fine by you? Is that your official standard?

Maybe the phrasing of the warning should be changed, to explain that copying one style is wrong, but copying more than one style is fine.

I am aware of at least one model that was marked with this warning, that copying much more than one style, but was marked seemingly just due to using the letters "S" "A" "M", how do you explain this fact? Do you have another special rule for models that use the letters "S" "A" "M"?

Another point, on the site, there are Textual Inversion embeddings for SD v 1.5 that point to the same style that was marked by the message above. It proves my point that all models are "guilty" of copying even this particular style, and you should apply the same rules for all models, including any model based on SD v 1.4, 1.5, 2.0, and 2.1, i.e. all models on the site, mark them all or mark none.

**Comment by a-l-e-x-d-s-9** [comment]
A couple of people brought up the argument of overfitting, my response:
1. Usually, even with the worst overfitting, you can NOT reproduce the exact original input image. But you are welcome to prove me wrong. Please show that all of those models are only generating the input images for empty or random prompts. Good luck with that.
2. Some of those models are well trained, without overfitting at all, why are you dismissing them all as the same?
3. At least one model of those that were marked, can barely reproduce the prohibited style, why was it marked too? Just for using the letters "S" "A" "M"? Are you fine with such standards?
4. On the site, there are Textual Inversion embeddings for SD v 1.5 that point to the same style that was marked by the message above. It proves my point that all models are "guilty" of copying even this particular style, and should receive the same warning, and possibly be deleted as the next step.
5. Let's read the message on the site again: "Someone has submitted a claim that this uses their art in it's training data and this claim is pending review". Do see here something about overfitting? The criterion is just to train a model of someone's images, all models are "guilty" of that.

**Reply by JustMaier** [reply]
1. Agreed. I've heard artists claiming that a model can replicate their work perfectly. I have yet to see that.
2. I don't have anything to add here.
3. Agreed, it seems the models that were reported were pretty much just random models that included Sam in the name, include a merge that doesn't look like the work of Sam Yang much at all...
4. If artists believe that their work should never be used as training data for a fine-tuned model meant to imitate their style, I'd imagine they'd report it even if it was a poor attempt.
5. The message is pretty generic. The intent is just to let users know that we've been asked to review the model with an Artist.

**Reply by DreamOfGit** [reply]
Keep me honest if I am wrong..

But I'm unclear on a few things.

- If you review a model and determine it has bias against a specific artist, will you then delete it?
- If they resubmit the removed model merged with another checkpoint - is it then okay?
- How exactly do you even verify what data-set was ingested to train that model.. ? Do you have a methodology to backwards resolve a checkpoint? _(Is this even scalable, does it have any potential false positives?)_
- Why is one model wrong but one with a more broad set of ingestion material totally fine? Does this **actually** have any meaningful impact on what damage(s) the artist perceives is being done? (If a broader model can achieve the same results - is there any value?)

I just don't understand how this would be even achievable in practice.

This feels like a _"nice thing to have"_ but will either result in nothing tangible happening or an "innocent model" _(some would argue they don't exist yet)_ being blown away as collateral damage.

Again - if my logic is off, tell me. 

The precedent this all sets in my mind- is that Civitai could at anytime decide to delete models / associated prompts & documentation - basically at a whim, based on a report and how well that report is vetted, which I don't understand the process behind.

This hurts Civitai's ability to be a 'source of truth' for information - AKA anything posted there now needs to be backed up in another location that is more resilient to "single purpose checkpoint" reporting.

EDIT: [deleted this section] Found the ToS (It gets pushed really far down when a lot of models load.. my mistake)

Thank you for reading if you made it this far. :)

**Reply by maxhulker** [reply]
@DreamOfGit to your points: 

_- If you review a model and determine it has bias against a specific artist, will you then delete it?_

**No the model is not deleted or removed, the point of this reporting system is to start a dialog, ideally with the artist. which is why we require the artist themselves to make the claim, not just anyone. The purpose of this dialog is to speak with artists about AI art and hopefully get them to make their own model, or work with the model creators who already have models up. We're well aware this is an optimistic goal, but it's our site and we're happy to test this out**

_- If they resubmit the removed model merged with another checkpoint - is it then okay?_

**See the above answer. really both ways are fine, in the unlikely event we do get contacted by an actual artist, have a discussion and they still insist we remove the model we'd probably advise this course of action to the model creator (or even just changing the name and description to not reference the artist) but it's not up to us to remove the model at that point, As we've stated before we won't remove anything from the site unless it breaks TOS. Come down to how generous the model creator wants to be**

_- How exactly do you even verify what data-set was ingested to train that model.. ? Do you have a methodology to backwards resolve a checkpoint? (Is this even scalable, does it have any potential false positives?)_

**Our methodology is looking at it, if your model explicitly states "trained from XYZ artist" then we don't really have to look very hard. Again though, we have NOTHING against models trained on specific artists, and with the above answers in mind understand that even if this artist does raise a claim against a model, the purpose is purely for dialog**


_- Why is one model wrong but one with a more broad set of ingestion material totally fine? Does this actually have any meaningful impact on what damage(s) the artist perceives is being done? (If a broader model can achieve the same results - is there any value?)_

**kinda the crux of the issue. Same with the idea that even if we or the model creator did remove the model, it would just pop up somewhere else, no killing this tech now. This is why we think it's important to speak to these artists, and this is the avenue we're giving them to reach out.**

the last point I want to address: 

_The precedent this all sets in my mind- is that Civitai could at any time decide to delete models / associated prompts & documentation - basically at a whim, based on a report and how well that report is vetted, which I don't understand the process behind._

I mean yes, we can. We could do that already without a report. This system does not increase the chances of that happening as we are really committed to NOT removing anything unless it breaks our TOS. (which we've tried to keep as close to legal and just common sense) 
The takeaway here is that based on an artist making a claim, legitimate or not, we will NOT remove a model.

**Reply by DreamOfGit** [reply]
Thank you for reading my comments and responding. 

I can see how it might of been painful to address each point when the response to the first point was "We won't delete models unless they break ToS". I've read your ToS and it seems perfectly sane & common sense to me. Great response to hear.

**My own personal commentary - Ignore if you want:**

I still think the warning message is ambiguous and given your team size (which was mentioned before) is this really something you want to burn velocity and time on right now - chasing down dubious reports for "investigation" which is basically trying to convince authors & model creators in an email to "play nice" - only to have them post the email out of context on twitter and have their fans bash your company & product?

This feels like lose-lose. 

Otherwise though, I wish you luck and I hope you prove me wrong.

**Reply by maxhulker** [reply]
Not painful at all, apologies if it came off that way, was going for a tone of sincerity. 

If the worst that comes out of this is that artists give our site free publicity, we're okay with that. Sure 95% of the people they're going to be posting it to will vocally side with them, but the fact remains that every time an artist posts about us our traffic and downloads surge. So pretty sure their fans are happy to try this tech out for themselves, which is pretty cool. 

I agree the warning message is ambiguous, even the idea that it's seen as a warning and not just a notification on the model is probably a good indicator we should rework the text there. 

On the optimistic side, (the less realistic side) if we can get even one artist to say "hey, this is pretty cool I should make a sanctioned model of my work" then we've contributed somewhat to the embracement of this technology. Otherwise, these folks are going to get steamrolled and as empathic people, we want to at least try to help them, even if they don't want to help themselves.

**Comment by JustMaier** [comment]
@a-l-e-x-d-s-9 

The issue of fine-tuning models on the works of a single artist raises complex questions about law, morality, and ethics. While we have tried to gather input from both sides of the debate, it has been difficult to find a clear answer that addresses the concerns of all parties involved. As a result, we have decided to approach this issue on a case-by-case basis until we can establish a precedence.

To facilitate this process, we recently added a feature that allows artists to report models that they believe were fine-tuned on their work. Our intention was to provide a way for artists to connect with us and work towards a resolution with the model creators. In the interest of transparency, we also chose to inform the community when such a report has been made.

Please note that our goal is only to remove models that violate our Terms of Service. We also want to be sensitive to the feelings of hurt expressed by creators who see their work being taken from them. This applies to both the artists whose styles are being imitated and the creators of the fine-tuned models.

In regards to the models that are currently flagged, it appears that they are a random assortment of models that include the name "Sam." We have reached out to Sam Yang and the individual who made the report, but have not yet received a response. If we do not hear back within 7 days, we will dismiss the claim.

We would like to know your thoughts on the following questions:

1.  Would you prefer that we not be transparent about claims made by artists regarding specific models?
2.  If you do want us to continue being transparent, is there a different way you would like us to display this information that might not be considered shameful?

**Reply by JustMaier** [reply]
I'd like to give this more thought and continue this. As you can probably assume, with such a small team and so many things to build, we're pretty backlogged.

**Comment by ashen-sensored** [comment]
Great... I just got my warning banner within 24 hours of my upload.
https://civitai.com/models/2706/mafuyu-diffusion
![Screenshot 1](https://user-images.githubusercontent.com/121544382/209784905-d7136801-506b-456f-a835-96cc281af498.png)

I must admit, it is really "encouraging" for a trainer knowing that the model you spent countless time and effort fine tunning/ messing with hyper params can be taken down at any moment.

![Screenshot 3](https://user-images.githubusercontent.com/121544382/209787979-450f522a-f284-4576-9019-c7be93554ddf.png)

As I clearly stated in my description, I didn't even "target trained" the artist's works. My dataset is scraped from public image sites with specific tags (literally the same training procedure as SD 1.4, 1.5, 2.0, 2.1). After the training, I found out that the model performs exceptionally well on specific prompt from tags of this artist's images and developed a heavy bias towards this artist's style, and that's why I gave it the name, not the other way around.

I don't know why even bother having a cloud based community anymore. So everyone must be gathering as much models as possible to avoid one random day losing access to a specific model?

If anyone is seeing this, check the link I posted above. I will leave this site and never come back if my model goes down, and you should too.

**Reply by a-l-e-x-d-s-9** [reply]
You should not be discouraged. I'm sure you invested many many hours to make a great model.
I don't know if CivitAI will exist for long, but I am sure that your work will exist, and will be appreciated.

**Reply by JustMaier** [reply]
Not sure if you read our various posts on this, but the whole reason we have this message is so that there is no chance for anything to "be taken down at any moment". When a claim is made we'll reach out to you and the artist making the claim to determine the appropriate action.

Speaking of which, it'd be great if we could connect with you. We'll be emailing you today 👍

**Comment by a-l-e-x-d-s-9** [comment]
@JustMaier

Thanks for your response. I wish only the best to CivitAI, I hope it would become a very successful business, with great user experience and clear rules.

.1. You are using the term "fine-tuning models" as if there are two completely different kinds of models, one type is a general model which can produce many styles and the other type is a fine-tuned model that can produce only a single specific style. I would argue that this dichotomy is false due to many reasons.

.1.1. Dreambooth models or regular continued training models, are not erasing most of the content from the base model, they are adding new concepts and styles. Most of the styles of the original model have remained there. So if the original model had X number of styles, the fine-tuned model has X+1 styles. Maybe some of the styles got distorted, nevertheless, the additional training does not erase all base styles. There is no fine-tuned model with a single style, unless it was trained from zero. I don't think you would be able to find even a single model that was created that way.

.1.2. Most of the Dreambooth models are using regularization images while training to minimize the bleeding of newly added content into existing areas. Practically the training process is designed to add a style/concept without erasing the existing style/concept. Therefore by design, fined-tuning does not do what the original dichotomy assumes.

.1.3. With most Dreambooth models you don't get the added style without using a triggering keyword. Just like any other concept. Obviously, the randomness of the system can push it more or less to certain areas.

.1.4. Fine-tuning can add more than one style at once to the base model, there are many examples of such models working as well as any other fine-tuned model. Is adding more than one style less fine-tuning due to this fact? When a single style is added it is bad, when two or more styles are added it is acceptable?! SD 1.4 and 1.5 are continued training of previous training themselves.

.2. I've already [addressed here](https://github.com/civitai/civitai/discussions/114#discussioncomment-4497687) the "overfitting" argument, so I want preemptively say that the overfitting argument is a bad argument. Not only that, overfitting can be a tool for mixing models, to steer easily the baseline of a model into the desired direction. Overfitting can be easily normalized and adjusted by mixing it with a low ratio with another model. Also, any model can be overfitted terribly by adding repeatedly its differences from the base to itself.  

.3. The issue of transparency is an important one, it's better for everybody to know what's happening. You definitely should continue the line of transparency. If the site would decide on a major change in policy without properly communicating it in advance it can ruin the reputation of the site and possibly devastate its potential future.

.4. You added an option for artist report models that don't violate your TOS. What is the use of it? 

.5. I have argued my points in the beginning, artists don't have a legal case against anyone, not the model creator and not against the site that distributes them. You would have been in legal procedure with them if they had legal justification. Currently, artists are harassing and bullying individuals who produce models that resemble their style. And you are helping them by letting them mark any model, because something resembles their style. The person who created the model has not violated the law or even your TOS, what's the point of adding a warning to the model? 

.6. All models can imitate many artists, if you believe that it's immoral. You need to add it to your TOS. Then remove all models.

.7. I don't understand why would you allow anyone to mark certain models with strange warnings. If there is a violation of TOS, you can add a small notice about the report inside the page of the model, with an option to read about the report, and maybe add a discussion section. But you definitely should not mark models when they are shown in the preview besides others, like some sort of badge of shame.

.8. It's nice that you sympathize with artists, we all do. It's a new technology and the world needs to adjust to the change. But some artists want to go back in time to before this technology existed, and it is just impossible. They will use anything to attack anyone on their way. If you are caving in, by scapegoating some invented bad guys that created "bad" fine-tuned models. And by repeating legally not based and immoral arguments regarding style ownership. You are setting yourself to failure. If you give up on just a few models, that did nothing worse than any other model, you will lose it all, because you lost any coherency and option of equal implementation of rules.

**Reply by maxhulker** [reply]
I can tell you're really impassioned about this, so first off, want to thank you for being so persistent in making sure your points are heard.  

I'm going to plainly state our position on this and why we've added this "claim" system, what its intention was/is, and how we've decided to clarify it going forward. 

The "This uses my art" reporting option is there to allow artists a single specific point to reach out and connect with us, rather than just a wash of models and no clear contact information, The intention is that once an artist has contacted us, we first verify that the person reporting is indeed who they claim to be.  Then we can start a dialog around AI art and offer to work with them to create an official model of their work. We don't expect many to go for this, but we think it's worth trying. 

To be clear, this in no way means a model will be removed. I cannot emphasize enough how strongly we feel about not removing models unless they conflict with TOS, and even then we've always reached out to model creators to ask them to make adjustments rather than just removing them. 

Going forward we'll still be collecting reports, however until we verify that the claim was made by the actual artist themselves, we will not be displaying the notification on the model page.

**Comment by Moebytes** [comment]


**Reply by rsadwick** [reply]
🦜 You are parroting misinformation, please do research into how things work:

There are plenty of high level explanations you can find anywhere within different communities.  You can also contribute and get a hands on experience with Stable Diffusion or other open source projects.  (https://github.com/CompVis/stable-diffusion)

Let's learn, create, embrace, and push this technology instead of stifling innovation.
